.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "airbnb",
   "rules": {
+    "linebreak-style" : "warn",
```

Comment:
I guess you shouldn't change this config here. Let's discuss it separately

Reply:
Ok